packages/identity-service/src/authMiddleware.js

```diff
@@ -38,6 +43,54 @@ const queryDiscprovForUserId = async (walletAddress, handle) => {
   }
 }
 
+/**
+ * queryDiscprovForUserId - Queries the discovery provider for the user w/ the walletaddress
```

Comment:
nit: this comment doesn't seem to make sense?

Reply:
doh!